### DISCUSSION TITLE
The Snap package should have a proper description.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [godotengine/godot-proposals#7763] [Platforms]
https://snapcraft.io/godot-3-1

> blah

**Comment by AThousandShips** [comment]
Are there current snap packages? That one is four years old and is for a version long out of support, the 3.1 package is not supported in any way

**Reply by Calinou** [reply]
Also, we don't officially maintain Snap packages for Godot. Please contact their upstream maintainers.

**Reply by roke-julian-lockhart** [reply]
@Calinou, it states that it's officially maintained, so perhaps it would be better for you to contact them at https://forum.snapcraft.io/.

**Reply by AThousandShips** [reply]
Where does it state that?

**Reply by Calinou** [reply]
> Where does it state that?

The author is the verified godot-developers organization, but I think this dates back years ago where we thought we'd have enough time available to maintain the Snap. This turned out not to be the case. The Flatpak already represents maintenance overhead on its own, and having a Snap on top of that would make things worse.
